GameStore.DataAccess/Repositories/Implementation/GameRepository.cs

```diff
@@ -1,31 +1,38 @@
 ﻿using GameStore.DataAccess.EntityModels;
 using System;
 using System.Collections.Generic;
+using System.Data.Entity;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace GameStore.DataAccess.Repositories.Implementation
 {
     public class GameRepository : IRepository<Game>
     {
-        private GameRepository()
+        private GameRepository(DbContext gameContext)
         {
-            gameContext = GameStoreContext.GetInstance();
+            this.gameContext = gameContext as GameStoreContext;
```

Comment:
Try to reduce using of `this.`. Better to give names which will reflect if the item is a parameter